rcl_interfaces/msg/IntegerRange.msg

```diff
@@ -0,0 +1,15 @@
+# Represents bounds and a step value for an integer typed parameter.
+
+# Lower bound for valid values, inclusive.
+int64 minimum_value
+
+# Upper bound for valid values, inclusive.
+int64 maximum_value
+
+# Size of valid steps between the lower and upper bound.
+# Step should be positive, and a step of zero implies a step size of one, i.e.
+# all integer values between the bounds are valid.
```

Comment:
Because, I don't even know how you'd represent that on a slider? The step is essentially capped at the distance between maximum and minimum, anything larger is effectively the same thing I guess, but it seems like something that is more likely to be an error than intentional.